packages/account/src/Components/sent-email-modal/sent-email-modal.jsx

```diff
@@ -78,7 +117,7 @@ const SentEmailModal = ({ identifier_title, is_open, onClose, onClickSendEmail }
         </SendEmailTemplate>
     );
 
-    if (isMobile()) {
+    if (isMobile() && !is_modal_on_mobile) {
```

Comment:
Added this props due to the responsive design which should be modal and not a fullscreen page.